Explorer/Assets/DCL/Character/CharacterMotion/Components/CharacterAnimationComponent.cs

```diff
@@ -4,19 +4,55 @@ namespace DCL.CharacterMotion.Components
 {
     public struct CharacterAnimationComponent
     {
-        // buffer values for animation states
-        [Serializable]
-        public struct AnimationStates
+        public AnimationStates States;
```

Comment:
I think it is time to get rid of `States`, we had other values within CharacterAnimationComponent in the past but now we only use this, so its probably better if we merge both structs. WDYT?

Reply:
Great proposal. Let's do it then on the next iteration, where I will also re-work animation blending.